### ISSUE TITLE
Add option to disable class/id shorthand

### ISSUE THREAD

**Issue by AngusMorton** [marko-js/prettier#27]
### Description

Make the class/id shorthand behaviour customisable to allow people to use the full-form class/id syntax but still use Prettier.

### Why

* The shorthand syntax is not obvious to new users.
* Can be noisy with the Tags API.
* It isn't always possible, for example, to use Tailwind responsive utilities with colons (e.g. `md:px-4`) in the short-hand, which leads to inconsistencies with syntax in the same file.

### Possible Implementation & Open Questions

We could add a config option, `markoClassAndIdShorthands: "always" |  "never"`. Where:
* `always` - Prefers the shorthand format where possible. Given `<div class=some-class />` -> `<div.some-class />` , however, class names that cannot be shorthanded remain in the class attribute e.g. `<div class="md:px-4" />`. This is the current behaviour and the default value so it's backwards compatible.
* `never` - Uses the full form everywhere. Given `<div.some-class />` -> `<div class=some-class />`

**Open Questions**
* Do we need/want an `auto` option that matches the convention of the file?
* Happy to workshop the config name and values.

### Is this something you're interested in working on?

Yes.

**Comment by DylanPiercey** [comment]
@AngusMorton were hesitant to add a config option here since it goes against the ethos of prettier.

We recently merged https://github.com/marko-js/prettier/pull/22 which uses the longhand form in html mode and the shorthand's in concise mode.

We are currently debating if we should attempt to just preserve whatever the user had. If we do go down that path there should probably be a bit of refactoring done on this module to use our new AST/CST parser which preserves more information about the source file (the current Marko ast has all of this class stuff normalized before we can print it).

**Comment by AngusMorton** [comment]
Yep, that's fair enough. I wanted #22 but tried to avoid jumping straight to it because it would be a breaking change. I didn't see #22 get shipped and was using an older version of @marko/prettier.

Thanks!
